.golangci.yml

```diff
@@ -5,25 +5,21 @@ linters:
 
   disable:
```

Comment:
I would like to suggest you configuring nolintlint (enabled here) to use require-explanations:true

The nolint you add have no comment, it would be better if you could force yourself to add them

Reply:
https://github.com/nobe4/gh-not/pull/234/commits/75b90c15d569c3fd664a97a4634f09eb4dd46259